packages/server/lib/mud/setupNetwork.ts

```diff
@@ -21,6 +23,7 @@ export async function setupNetwork() {
     syncThread: "main",
     storeConfig,
     worldAbi: IWorld__factory.abi,
+    syncStoreCache: false
```

Comment:
node has some issue with `storeCache`, best to just disable it and rely on the `recs` sync